wrappers/javascript/indy-vdr-react-native/package.json

```diff
@@ -54,12 +53,5 @@
   "peerDependencies": {
     "react": ">= 16",
     "react-native": ">= 0.66.0"
-  },
-  "binary": {
```

Comment:
Won't this break the non-alpha releases? My preference would really be to fetch the libraries from gh in a post install (what node-pre-gyp does), but if there are benefits I am okay with it.

Reply:
we can go back to fetch when/if we can figure out publishing alpha releases of the rust libraries. I was thinking on maybe publishing alphas builds to GitHub Package Registry (maybe as maven artifacts)

Reply:
> I got rid of this because the new github actions copy the compiled binaries into the native folder, so the alpha release will target the latest binaries

but it only does this in the `alpha-release` action, correct? Normal releases will not contain the copied library and fail as we also do not push them to the repository.

Reply:
Honestly i don't see a reason to change this setup?

It works cross platform for Node.JS / iOS / Android and means you only download the binary for your platform (in the Node.JS release)

We've put quite some thought into this process and I would be really disappointed to see it replaced by something else that caters better to your use case (react native), but doesn't take into account Node.Js

Reply:
See my suggestion on how to approach this in the other comment